README.md

```diff
@@ -53,15 +55,47 @@
     This should generate new file at the location:
 
     ```
-    terraform/clusters/cluster1.run-sandbox.aws.ext.govsvc.uk/cluster.tf
+    terraform/clusters/${CLUSTER}.${AWS_ACCOUNT_NAME}.${CLOUD}.ext.govsvc.uk/cluster.tf
     ```
 
     This leaves you with a manual step of:
 
     ```sh
-    export DOMAIN=cluster1.gds-re-run-sandbox.aws.ext.govsvc.uk
+    export DOMAIN=${CLUSTER}.${AWS_ACCOUNT_NAME}.${CLOUD}.ext.govsvc.uk
     cd terraform/clusters/${DOMAIN}
-    aws-vault exec run-sandbox -- docker run -it --env AWS_DEFAULT_REGION --env AWS_REGION --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --env AWS_SESSION_TOKEN --env AWS_SECURITY_TOKEN --env DOMAIN --volume=$(pwd)/../../../:/terraform -w /terraform/terraform/clusters/${DOMAIN} govsvc/terraform {init, plan, apply}
+
+    aws-vault exec run-sandbox -- docker run -it \
+      --env AWS_DEFAULT_REGION \
+      --env AWS_REGION \
+      --env AWS_ACCESS_KEY_ID \
+      --env AWS_SECRET_ACCESS_KEY \
+      --env AWS_SESSION_TOKEN \
+      --env AWS_SECURITY_TOKEN \
+      --env DOMAIN \ --volume=$(pwd)/../../../:/terraform \
+      -w /terraform/terraform/clusters/${DOMAIN} \
+      govsvc/terraform init
+
+    aws-vault exec run-sandbox -- docker run -it \
```

Comment:
With the latest version of terraform you don't need to do `plan` action as it's included in the `apply` anyway. Takes room with new multiline, separate commands approach. We could make the scrolling smaller?

Reply:
will remove